plugins/xterm/xterm.plugin.bash

```diff
@@ -12,22 +12,22 @@ _omb_module_require plugin:bash-preexec
 
 function _omb_plugin_xterm_short_dirname {
   local dir_name=${PWD/~/\~}
-  if [[ ${OMB_PLUGIN_XTERM_SHORT_TERM_LINE-} == true ]] && ((${#dir_name} > 8)); then
+  if [[ ${OMB_PLUGIN_XTERM_SHORT_TERM_LINE:-} == true ]] && ((${#dir_name} > 8)); then
     dir_name=${dir_name##*/}
   fi
   echo "$dir_name"
 }
 
 function _omb_plugin_xterm_short_command {
   local input_command="$*"
-  if [[ ${OMB_PLUGIN_XTERM_SHORT_TERM_LINE-} == true ]] ((${#input_command} > 8)); then
+  if [[ ${OMB_PLUGIN_XTERM_SHORT_TERM_LINE:-} == true ]] && ((${#input_command} > 8)); then
     input_command=${input_command%% *}
   fi
   echo "$input_command"
 }
 
 function _omb_plugin_xterm_set_title {
-  local title=${1-}
+  local title=${1:-xterm}
```

Comment:
Not sure then if this needed as well. I though default `xterm` or `terminal` would be nice.

Reply:
I find `xterm` or `terminal` neutral. In case of the missing title the user at least knows which GUI application they look at.

Reply:
The problem with `xterm` is that the `xterm` plugin is used not only with xterm but also used with other terminals that support XTerm's sequences, so I'd use `terminal` if I choose one from `xterm` and `terminal`.